tests/Aspire.Hosting.Tests/ManifestGenerationTests.cs

```diff
@@ -374,10 +374,7 @@ public void EnsureAllAzureOpenAIManifestTypesHaveVersion0Suffix()
         var resources = publisher.ManifestDocument.RootElement.GetProperty("resources");
 
         var openai = resources.GetProperty("openai");
-        Assert.Equal("azure.openai.account.v0", openai.GetProperty("type").GetString());
-
-        var deployment = resources.GetProperty("deployment");
-        Assert.Equal("azure.openai.deployment.v0", deployment.GetProperty("type").GetString());
+        Assert.Equal("azure.bicep.v0", openai.GetProperty("type").GetString());
```

Comment:
Can you change this test to verify the manifest like the following?

https://github.com/dotnet/aspire/blob/525c59ad9d58de54c71071e248b060061dfecd56/tests/Aspire.Hosting.Tests/SqlServer/AddSqlServerTests.cs#L123-L153